about.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
+  <title>Furahia Insurance</title>
+</head>
+
+<body>
+  <header>
+    <div class="upper-mennu">
```

Comment:
- [ ] As requested by the previous reviewer, great job on changing the color of the top menu, however, there is a space around the top navigation that is making the design not look good, kindly fix it.

Your design
![image](https://user-images.githubusercontent.com/81932478/169654458-8e250e73-5615-4576-b6da-c1c6b2fd1adc.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/169654522-d174a18c-2c50-4a6c-8523-648ef50bf667.png)

Reply:
@Munsa1 , This is now fixed